deployments/puppet/README.md

```diff
@@ -48,7 +48,7 @@ This class accepts the following parameters:
 | `with_fluentd` | Whether to install/manage fluentd and dependencies for log collection.  The dependencies include [capng_c](https://github.com/fluent-plugins-nursery/capng_c) for enabling [Linux capabilities](https://docs.fluentd.org/deployment/linux-capability), [fluent-plugin-systemd](https://github.com/fluent-plugin-systemd/fluent-plugin-systemd) for systemd journal log collection, and the [required libraries/development tools](https://github.com/signalfx/splunk-otel-collector/blob/main/docs/getting-started/linux-installer.md#fluentd-configuration). | `true` |
 | `fluentd_config_source` | Source path to the fluentd config file.  This file will be copied to the `$fluentd_config_dest` path on the node.  See the [source attribute](https://puppet.com/docs/puppet/latest/types/file.html#file-attribute-source) of the `file` resource for supported value types.  The default source file is provided by the collector package. | `file:///etc/otel/collector/fluentd/fluent.conf` |
 | `fluentd_config_dest` | Destination path to the fluentd config file on the node. | `/etc/otel/collector/fluentd/fluent.conf` |
-| `manage_repo` | **Linux only**: In cases where the collector and fluentd apt/yum repositories are managed externally, set this to `false` to disable management of the repositories by this module.  The repositories should provide the `splunk-otel-collector` and `td-agent` packages.  **Note:** If set to `false`, the externally managed repositories should provide the `splunk-otel-collector` and `td-agent` packages.  Also, the apt (`/etc/apt/sources.list.d/splunk-otel-collector.list`, `/etc/apt/sources.list.d/splunk-td-agent.list`) and yum (`/etc/yum.repos.d/splunk-otel-collector.repo`, `/etc/yum.repos.d/splunk-td-agent.repo`) repository definition files will be deleted if they exist in order to avoid any conflicts. | `true` |
+| `manage_repo` | **Linux only**: In cases where the collector and fluentd apt/yum repositories are managed externally, set this to `false` to disable management of the repositories by this module.  **Note:** If set to `false`, the externally managed repositories should provide the `splunk-otel-collector` and `td-agent` packages.  Also, the apt (`/etc/apt/sources.list.d/splunk-otel-collector.list`, `/etc/apt/sources.list.d/splunk-td-agent.list`) and yum (`/etc/yum.repos.d/splunk-otel-collector.repo`, `/etc/yum.repos.d/splunk-td-agent.repo`) repository definition files will be deleted if they exist in order to avoid any conflicts. | `true` |
```

Comment:
Same comment here. Let's move to another PR and have clean git history :)

Reply:
Moved to #483